instructor/ui/sketch_tool.js

```diff
@@ -129,7 +131,8 @@ function draw_line(x1 ,y1 ,x2 ,y2, color){
 
 }
 
-function nameXAxis(condition){
+function nameXAxis(condition, this_paper){
+    paper = this_paper;
```

Comment:
Is this meant to be a local variable? If so we should make this `var paper = this_paper;`. Otherwise this function redefines the global `paper` which is usually not a good idea